signac/contrib/project.py

```diff
@@ -762,17 +1080,19 @@ def write_statepoints(self, statepoints=None, fn=None, indent=2):
 
         If the file already contains statepoints, all new statepoints
         will be appended, while the old ones are preserved.
+        See also :meth:`dump_statepoints`.
```

Comment:
Use the "See Also" section: https://numpydoc.readthedocs.io/en/latest/format.html#sections